search/search_service/config.py

```diff
@@ -58,3 +58,30 @@ class LocalConfig(Config):
         'title': 'Search Service',
         'uiversion': 3
     }
+
+
+class AwsSearchConfig(LocalConfig):
+    import boto3
```

Comment:
in terms of docs I think we could just add a docstring to this class. correct me if I am wrong but `AwsSearchConfig` is prepared for one of many possible deployment strategies of amundsen on AWS - the one where you use AWS Elasticsearch service and authenticate containers to AWS ES service using IAM roles. The docstring should be explicit about the fact that it's only for this use-case.

Reply:
@mgorsk1 I've added docstring; please check if it's fine.

Reply:
perfect, thanks!